api/src/main/java/org/pmiops/workbench/workspaces/resources/UserRecentResourceServiceImpl.java

```diff
@@ -72,24 +82,20 @@ public void updateConceptSetEntry(long workspaceId, long userId, long conceptSet
 
   @Override
   public void updateDataSetEntry(long workspaceId, long userId, long dataSetId) {
-    if (workbenchConfigProvider.get().featureFlags.enableDSCREntryInRecentModified) {
-      updateUserRecentlyModifiedResourceEntry(
-          workspaceId,
-          userId,
-          DbUserRecentlyModifiedResource.DbUserRecentlyModifiedResourceType.DATA_SET,
-          String.valueOf(dataSetId));
-    }
+    updateUserRecentlyModifiedResourceEntry(
+        workspaceId,
+        userId,
+        DbUserRecentlyModifiedResource.DbUserRecentlyModifiedResourceType.DATA_SET,
+        String.valueOf(dataSetId));
   }
 
   @Override
   public void updateCohortReviewEntry(long workspaceId, long userId, long cohortReviewId) {
```

Comment:
Could updateCohortEntry and updateCohortReviewEntry be combined into a single function? It looks like they call the same underlying function with only an enum parameter different. Maybe we could pass an enum to the combined function.

Reply:
I'm not sure which method signature is best - but if we did combine these, we could apply this to all 4 data-tab resources, and do the same to the delete methods.

Reply:
I really like that suggestion, but is it ok if i can do that in next PR. I can create a new story/sub-task for the same